services/galley/src/Galley/API/Federation.hs

```
@@ -279,7 +279,7 @@ onConversationUpdated requestingDomain cu = do
       SConversationMessageTimerUpdateTag -> pure (Just sca, [])
       SConversationReceiptModeUpdateTag -> pure (Just sca, [])
       SConversationAccessDataTag -> pure (Just sca, [])
-
+      SConversationSelfInviteTag -> pure (Nothing, []) -- TODO(elland): Should not happen. Should we throw?
```

For now it might make more sense to throw a 403 instead of an empty list.

---

Actually this is federation, we don't throw errors.